docs/notebooks/governance_integration.ipynb

```diff
@@ -385,7 +385,7 @@
     "Instead of Lens automating the communication between your assessment environment and governance, you will instead have to download assets and upload them. \n",
     "\n",
     "### Getting the assessment spec\n",
-    "The asset that you may have to get _from_ the governance app is the assessment spec. This can be found under the \"Technical Requirements\" of your Use Case. Once you download the assessments spec, you can read it with a CredoGovernance object:\n",
+    "The asset that you may have to get _from_ the governance app is the assessment spec. This can be found under your use case's \"Technical Requirements\" tab inside \"Align\". Once you download the assessments spec, you can read it with a CredoGovernance object:\n",
```

Comment:
```suggestion
    "The asset that you may have to get _from_ the governance app is the assessment spec. This can be found under your use case's \"Risk Assessment\" tab inside \"Align\". Once you download the assessments spec, you can read it with a CredoGovernance object:\n",
```